aries-site/src/examples/templates/wizard/components/StepContent.js

```diff
@@ -1,19 +1,48 @@
 import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
 import { Box, Form, ResponsiveContext } from 'grommet';
 import { StepHeader, WizardContext } from '.';
 
-export const StepContent = () => {
+export const StepContent = ({ onSubmit }) => {
   const size = useContext(ResponsiveContext);
   const {
     activeIndex,
+    setActiveIndex,
     formValues,
     id,
     ref,
     setFormValues,
+    setValid,
     steps,
     width,
   } = useContext(WizardContext);
 
+  const handleSubmit = (evt) => {
```

Comment:
```suggestion
  const handleSubmit = (event) => {
```
Small suggestion but just for consistency across our other examples.

Reply:
Done